README.md

```diff
@@ -7,13 +7,27 @@
 
 # RiotX Android
 
-RiotX is an Android Matrix Client currently in development. The application is not yet available on the PlayStore. 
+RiotX is an Android Matrix Client currently in beta but in active development.
 
-It's based on a new Matrix SDK, written in Kotlin.
+It is a total rewrite of [Riot-Android](https://github.com/vector-im/riot-android) with a new user experience. RiotX will become the official replacement as soon as all features are implemented.
 
-Download nightly build here: [![Buildkite](https://badge.buildkite.com/657d3db27364448d69d54f66c690f7788bc6aa80a7628e37f3.svg?branch=develop)](https://buildkite.com/matrix-dot-org/riotx-android/builds?branch=develop)
+[<img src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png" alt="Get it on Google Play" height="60">](https://play.google.com/store/apps/details?id=im.vector.riotx)
+
+
+# New Android SDK
+
+RiotX is based on a new Android SDK fully written in Kotlin (like RiotX). In order to make the early development as fast as possible, RiotX and the new SDK currently share the same git repository. We will make separate repos once the API is stable enough.
+
+
+# Roadmap
+
+The current target is to release an application out of beta with the same level of features (and even more) as Riot.
+The roadmap has 3 phases:
+
+- [phase 0](https://github.com/vector-im/riotX-android/labels/phase0): Prototyping / Project setup
+- [phase 1](https://github.com/vector-im/riotX-android/labels/phase1): Beta release to the Play Store
+- [phase 2](https://github.com/vector-im/riotX-android/labels/phase2): Out of beta
 
-Matrix Room: [![RiotX Android Matrix room #riot-android:matrix.org](https://img.shields.io/matrix/riotx:matrix.org.svg?label=%23RiotX:matrix.org&logo=matrix&server_fqdn=matrix.org)](https://matrix.to/#/#riotx:matrix.org)
```

Comment:
no more advertising on RiotX room?

Reply:
There is already a link in the header.
Hmm. We could move the link in the Contributing section, no?

Reply:
I do not see the link in the header.
But yes, it is also in the CONTRIBUTING.md file